frontend/src/metabase/dashboard/components/AddSeriesModal/AddSeriesModal.tsx

```diff
@@ -144,6 +145,9 @@ export class AddSeriesModal extends Component<Props, State> {
             <Visualization
               canRemoveSeries={CAN_REMOVE_SERIES}
               className="spread"
+              errorView={
```

Comment:
Can we instead of overriding the component pass the message itself, e.g. e.g. `error={t`Unable to combine these questions`}`?

Reply:
20b605a9a86b62268645991776f276fca15bf070

Reply:
Refactored `errorView` prop to `errorMessageOverride`: 20b605a9a86b62268645991776f276fca15bf070